### PR TITLE
Add simple `python` client example

### PR DESCRIPTION
**Pull request by wslulciuc**
This PR adds a simple `python` client example to outline usage. This PR also fixes a bug where if `mark_as_running` was set to `True`, an exception was thrown and now uses the `runID` present in _create run_ API response.

### CODE REVIEW

File: clients/python/examples/simple.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+marquez_client.create_dataset(
+    namespace_name="my-namespace",
+    dataset_type=DatasetType.DB_TABLE,
+    dataset_name="my-dataset", # You can also use the physical name as the dataset name.
+    dataset_physical_name="public.mytable",
+    source_name="my-source",
+    fields=[
+        {'name': 'field0', 'type': 'INTEGER'},
+        {'name': 'field1', 'type': 'INTEGER'},
+        {'name': 'field2', 'type': 'STRING'}
```
Comment (fm100) [inline clients/python/examples/simple.py:54]: Very minor, but can we use double quotation here for the sake of consistency?

File: clients/python/examples/simple.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+# Now, let's create a run for 'my-job' and capture any runtime arguments:
+active_run = marquez_client.create_job_run(
+    namespace_name="my-namespace",
+    job_name="my-job",
+    mark_as_running=True
+)
+
+# ID of active run to mark as 'COMPLETE'.
+run_id = active_run['id']
```
Comment (fm100) [inline clients/python/examples/simple.py:79]: Same for here, double quotation for consistency sake.

File: clients/python/marquez_client/client.py [hunk 3]
```diff
@@ -303,7 +303,7 @@ def create_job_run(self, namespace_name, job_name, run_id=None,
             payload=payload)
 
         if mark_as_running:
-            response = self.mark_job_run_as_started(run_id)
+            response = self.mark_job_run_as_started(response['id'])
```
Comment (fm100) [inline clients/python/marquez_client/client.py:306]: Same here too.

**Review by fm100 — APPROVED** [review]
With very minor comments, LGTM.

File: clients/python/examples/simple.py (hunk 1 above)
Comment (wslulciuc) [reply clients/python/examples/simple.py:54]: Sure, but most of the `py` client code uses single quotes, so I'll default to that 👍

**Pull request merged**